packages/babel-compat-data/scripts/download-compat-table.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-COMPAT_TABLE_COMMIT=ca2d70e305bc3295ade6b06a1c33a929db1ccacf
+COMPAT_TABLE_COMMIT=e03b35f64d082382da113b163728a67f749a9b78
```

Comment:
```suggestion
COMPAT_TABLE_COMMIT=bcf74805f08ef13371ebe3aa36030bed4cd9a902
```
This commit includes Node 16 support.